src/main/java/app/gpx_animator/Renderer.java

```diff
@@ -414,13 +445,7 @@ private void keepLastFrame(final RenderingContext rc, final FrameWriter frameWri
             final Point2D marker = drawMarker(bi, frames);
             if (font != null) {
                 drawInfo(bi, frames, marker);
-
-                String att = resourceBundle.getString("configuration.attribution");
-                if (cfg.getAttribution().equals(att)) {
-                    drawAttribution(bi, att.replace("%APPNAME_VERSION%", Constants.APPNAME_VERSION).replace("%MAP_ATTRIBUTION%", ""));
-                } else {
-                    drawAttribution(bi, cfg.getAttribution());
-                }
+                drawAttribution(bi, cfg.getAttribution());
```

Comment:
Looks like you removed the fix of bug #226. I guess this was a merge conflict that was resolved the wrong way.